### DISCUSSION TITLE
How to specify program name seen in windows taskbar settings

### DISCUSSION THREAD

**Discussion by sonnin** [HavenDV/H.NotifyIcon#135] [Q&A] [answered]
Hi,

I am creating a WinUI application with a TrayIcon like the H.NotifyIcon.Apps.WinUI example.
When I right click on the windows taskbar and select "Taskbar settings" I can see the program under "Other system tray icons" and select if the TrayIcon for the program are to be show or hidden.

In the H.NotifyIcon.Apps.WinUI example the name of the program listed is "H.NotifyIcon.Apps.WinUI".

My question are how do I change this name ?

Thanks Sonni

**Comment by HavenDV** [comment]
As far as I remember, it takes the name from Package.appxmanifest

**Reply by sonnin** [reply]
I have tried to change all the H.NotifyIcon.Apps.WinUI references in the Package.appxmanifest, but this does not appear to change program name shown in windows settings.

Do you know which tag in the Package.appxmanifest that sets the program name ?

**Reply by HavenDV** [reply, marked as answer]
May be just Assembly Name/Exe file name? Try to rename project or use `<AssemblyName>YourName</AssemblyName>` in your properties in .csproj file.
Because I'm not sure about unpackaged WinUI apps, it does not use Package.appxmanifest.

**Reply by sonnin** [reply]
Thanks, <AssemblyName> is what is shown in windows taskbar settings
